test/main.spec.ts

```diff
@@ -14,7 +14,12 @@ it('has config', () => {
 
 it('checks path', () => {
   PathUtils.checkExists = jest.fn().mockResolvedValue(true);
-  PathUtils.checkExists(PathUtils.savePath());
+  PathUtils.savePath().then((path) => {
+    PathUtils.checkExists(path);
+  }).catch((err) => {
+    throw err;
+  });
```

Comment:
couldn't this be done with await instead of then?

Reply:
Certain PathUtils functions have been converted to asynchronous, when attempting to await functions an error emerges as follows:
Top-level 'await' expressions are only allowed when the 'module' option is set to 'esnext' or 'system', and the 'target' option is set to 'es2017' or higher.ts(1378)